pkgs/top-level/all-packages.nix

```
@@ -22727,6 +22727,8 @@ with pkgs;
 
   freeradius = callPackage ../servers/freeradius { };
 
+  freshrss = callPackage ../servers/freshrss { };
```

Shouldn't it be here instead?
```suggestion
  freshrss = callPackage ../servers/web-apps/freshrss { };
```